apps/e2e/forms-storybook/src/app/checkbox/checkbox.component.html

```diff
@@ -80,6 +80,34 @@
       </sky-checkbox>
     </sky-column>
   </sky-row>
+  <sky-row class="sky-padding-even-md">
+    <sky-column>
+      <sky-checkbox
+        labelText="Checkbox with open easy mode help inline"
+        helpPopoverContent="popover Content"
+      >
+      </sky-checkbox>
+    </sky-column>
+  </sky-row>
+  <sky-row class="sky-padding-even-md">
+    <sky-column>
+      <sky-checkbox
+        labelText="Checkbox with easy mode help inline"
+        helpPopoverContent="popover Content"
+      >
+      </sky-checkbox>
+    </sky-column>
+  </sky-row>
+  <sky-row class="sky-padding-even-md">
+    <sky-column>
+      <sky-checkbox
+        labelText="Required checkbox with easy mode help inline"
+        helpPopoverContent="popover Content"
+        [required]="true"
+      >
+      </sky-checkbox>
+    </sky-column>
+  </sky-row>
```

Comment:
I think it would also be good to have a screenshot of a required checkbox with inline help (like you have) but also with an error message to make sure all the pieces work well together.

Like this one, but with inline help:
![Screenshot 2024-04-19 at 12 01 45 PM](https://github.com/blackbaud/skyux/assets/12497062/98b93985-2f87-4b1a-a915-28a0e191669d)